src/index.html

```diff
@@ -11,7 +11,161 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal Information</legend>
+        <label class="form_field">
+          Surname :
+          <input
+            name="Surname"
+            type="text"
+            autocomplete="off"
+            required
+            minlength="5"
+            maxlength="35"
+          >
+        </label>
+        <label class="form_field">
+          Name :
+          <input
+            name="Name"
+            type="text"
+            autocomplete="off"
+            required
+            minlength="5"
+            maxlength="25"
+          >
+        </label>
+        <label class="form_field">
+          How old are you?:
+          <input
+          name="Age"
+          id="Age"
+            type="number"
+            required
+            minlength="1"
+            maxlength="100"
+            value="12"
+          >
+        </label>
+        <label class="form_field">
+          Full date of birth:
+          <input
+            name="submissionDate"
+            type="date"
+            required
+          >
+        </label>
+        <label class="form_field">
+          I accept the terms of agreement
+          <input
+            type="checkbox"
+            name="Terms"
+            id="terms"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="form_field">
+          Email:
+          <input
+            name="Email"
+            type="email"
+            required
+            minlength="10"
+            maxlength="35"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="form_field">
+          Password :
+          <input
+            name="Password"
+            type="password"
+            required
+            minlength="5"
+            maxlength="25"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An intersting fact about you!</legend>
+        <label class="form_field" >
+          Dou you love Cats?
```

Comment:
Fix labels here. It doesn't work correctly. Watch the video below 

https://github.com/mate-academy/layout_html-form/assets/99403198/98f58edb-9a9f-40a3-9e0b-67f74241a8e2